client/app/license/order/license-order.html

```diff
@@ -194,100 +194,92 @@ <h2 data-translate="license_order_step3_title"></h2>
-                                    data-ng-if="!agoraEnabled"
-                                    data-translate="license_order_step4_contracts_ask">
+                                    data-ng-if="!agoraEnabled && durations.details[selected.duration].contracts.length == 0"
+                                    data-translate="license_order_step4_question">
                             </strong>
                             <strong class="text-primary"
-                                    data-ng-if="agoraEnabled"
-                                    data-translate="license_order_step4_contracts_agora_ask">
+                                    data-ng-if="agoraEnabled && durations.details[selected.duration].contracts.length == 0"
+                                    data-translate="license_order_step4_clic_here">
                             </strong>
-                            <span data-ng-repeat="contract in durations.details[selected.duration].contracts track by $id">
-                                <a target="_blank"
-                                   rel="noopener"
-                                   class="d-inline-block oui-link_icon"
-                                   data-ng-href="{{ ::contract.url }}">
-                                   <span data-translate="common_newtab" class="sr-only"></span>
-                                   <span data-ng-bind="contract.url"></span>
-                                   <span class="oui-icon oui-icon-external_link" aria-hidden="true"></span>
-                                 </a>
-                            </span>
-                            <div class="checkbox">
-                                <label>
-                                    <input type="checkbox"
-                                           data-ng-model="contractsValidated.value" />
-                                    <span data-translate="license_order_step4_contracts_check"></span>
-                                </label>
+
+                            <div data-ng-if="!agoraEnabled && durations.details[selected.duration].contracts.length > 0">
+                                <strong class="text-primary"
+                                        data-ng-if="!agoraEnabled"
+                                        data-translate="license_order_step4_contracts_ask">
+                                </strong>
+                                <strong class="text-primary"
+                                        data-ng-if="agoraEnabled"
+                                        data-translate="license_order_step4_contracts_agora_ask">
+                                </strong>
+                                <span data-ng-repeat="contract in durations.details[selected.duration].contracts track by $id">
+                                    <a target="_blank"
+                                       rel="noopener"
+                                       class="d-inline-block oui-link_icon"
```

Comment:
```suggestion
                                       class="oui-link_icon"
```
looks like it's not needed

Reply:
Its required to make the contract links appear in new line.